src/internal.c

```diff
@@ -24668,6 +24668,14 @@ int SendData(WOLFSSL* ssl, const void* data, int sz)
         if (IsEncryptionOn(ssl, 1) || ssl->options.tls1_3)
             outputSz += cipherExtraData(ssl);
 
+#if defined(WOLFSSL_DTLS) && defined(WOLFSSL_DTLS_CID)
+        if (ssl->options.dtls) {
+            unsigned int cidSz = 0;
+            if (wolfSSL_dtls_cid_get_tx_size(ssl, &cidSz) == WOLFSSL_SUCCESS)
```

Comment:
NIT: Dtls13GetCidTxSize can return 0 and it doesn't need macros

Reply:
Yes but its in a different file. Do you want me to pull it out of `dtls13.c` and make it non-static?